openerp/addons/base/migrations/9.0.1.3/noupdate_changes.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8'?>
+<openerp>
+    <data>
+        <record model="res.groups" id="group_user">
+            <field name="implied_ids" eval="[(4, ref('group_no_one'))]"/>
```

Comment:
this will turn on the technical setting for every user. What's the rationale here as opposed to assign this to groyp_system?